index.bs

```diff
@@ -3470,6 +3471,15 @@ nothrow>WritableStreamDefaultControllerGetDesiredSize ( <var>controller</var> )<
   1. Return _controller_.[[strategyHWM]] − _queueSize_.
 </emu-alg>
 
+<h4 id="writable-stream-default-controller-should-delay-close-update"
```

Comment:
As noted in https://github.com/whatwg/streams/pull/672#issuecomment-275084660, spec is not yet sync with the impl. But thanks for review. I'll make sure that this gets fixed.